From 9443a247d78d7820eed84383f64dfb9a8b4f0970 Mon Sep 17 00:00:00 2001
From: Gary Buhrmaster <gary.buhrmaster@gmail.com>
Date: Tue, 14 Jul 2020 15:05:21 +0000
Subject: [PATCH 2/7] Rename the RecordingType stream selection UI element in
 recordingprofile.cpp

Renamed to avoid (global) name conflict with RecordingType enum in
libs/libmyth/recordingtypes.h
---
 mythtv/libs/libmythtv/recordingprofile.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mythtv/libs/libmythtv/recordingprofile.cpp b/mythtv/libs/libmythtv/recordingprofile.cpp
index 3434109437..55489718d7 100644
--- a/mythtv/libs/libmythtv/recordingprofile.cpp
+++ b/mythtv/libs/libmythtv/recordingprofile.cpp
@@ -1221,10 +1221,10 @@ class TranscodeLossless : public MythUICheckBoxSetting, public CodecParamStorage
     };
 };
 
-class RecordingType : public MythUIComboBoxSetting, public CodecParamStorage
+class RecordingTypeStream : public MythUIComboBoxSetting, public CodecParamStorage
 {
   public:
-    explicit RecordingType(const RecordingProfile &parent) :
+    explicit RecordingTypeStream(const RecordingProfile &parent) :
         MythUIComboBoxSetting(this), CodecParamStorage(this, parent, "recordingtype")
     {
         setLabel(QObject::tr("Recording Type"));
@@ -1633,7 +1633,7 @@ void RecordingProfile::CompleteLoad(int profileId, const QString &type,
     }
     else if (type.toUpper() == "DVB")
     {
-        addChild(new RecordingType(*this));
+        addChild(new RecordingTypeStream(*this));
     }
 
     if (CardUtil::IsTunerSharingCapable(type))
-- 
2.26.2

